### DISCUSSION TITLE
Command line way of switching to an existing preset (eg for a mouse)?

### DISCUSSION THREAD

**Discussion by Ultome** [sezanzeb/input-remapper#850] [General]
Input Remapper is amazing. I use it to change what my 5 buttons mouse buttons do when I play video games.
However, it would be even better if I could activate my "game" mouse layout from the command line (I do a bunch of things to set up my laptop for gaming and I'm trying to gather them all in a bash script to save some trouble — I switch a lot of times in a day).
So, is there a way to do this? I have a peset for my mouse layout, I just need to activate it. I'm on Manjaro Linux, with KDE Plasma (5).
Thanks!

**Comment by sabudum** [comment]
input-remapper-control --command start --device "USB Optical Mouse" --preset "my_awesome_preset"

Obviously you need to change the device name for your own mouse,and your preset name as well.

**Reply by Ultome** [reply]
Thanks a lot!

**Comment by RedSnt** [comment]
https://github.com/DreadPirateLynx/input-remapper-xautopresets seemingly switches your input-remapper device and preset based on what you set and for different windows. Seems like a good solution as well.  
I unfortunately can't use it as I use a SysV distro without a full SystemD installation, but Manjaro is all SystemD so should be a fit for you.
